xml/admin_ceph_gateway.xml

```diff
@@ -2692,349 +2692,279 @@ rgw_override_bucket_index_max_shards = 12
+<screen>&prompt.cephuser;radosgw-admin zonegroup delete --rgw-zonegroup=default</screen>
+   <para>Delete the default pools in your &ceph; storage cluster if they exist:</para>
+   <important>
+     <para>The following step assumes a multi-site configuration using newly
+       installed systems that are not currently storing data. <emphasis role="bold">Do not delete</emphasis>
+       the default &zonegroup; if you are already using it to store data.</para>
+   </important>
+<screen>&prompt.cephuser;ceph osd pool rm default.rgw.control default.rgw.control --yes-i-really-really-mean-it
+&prompt.cephuser;ceph osd pool rm default.rgw.data.root default.rgw.data.root --yes-i-really-really-mean-it
+&prompt.cephuser;ceph osd pool rm default.rgw.gc default.rgw.gc --yes-i-really-really-mean-it
+&prompt.cephuser;ceph osd pool rm default.rgw.log default.rgw.log --yes-i-really-really-mean-it
+&prompt.cephuser;ceph osd pool rm default.rgw.meta default.rgw.meta --yes-i-really-really-mean-it</screen>
+</sect3>
    <sect3 xml:id="ceph-rgw-fed-masterzone-createuser">
     <title>Creating System Users</title>
     <para>
-     To access zone pools, you need to create a system user. Note that you will
-     need these keys when configuring the secondary zone as well.
+     The <systemitem class="daemon">ceph-radosgw</systemitem> daemons must authenticate before pulling realm and
+     period information. In the master zone, create a system user to
+     facilitate authentication between daemons:
     </para>
 <screen>&prompt.cephuser;radosgw-admin user create --uid=zone.user \
---display-name="Zone User" --access-key=<replaceable>$SYSTEM_ACCESS_KEY</replaceable> \
---secret=<replaceable>$SYSTEM_SECRET_KEY</replaceable> --system</screen>
-   </sect3>
+--display-name="Zone User" --access-key=<replaceable>SYSTEM_ACCESS_KEY</replaceable> \
+--secret=<replaceable>SYSTEM_SECRET_KEY</replaceable> --system</screen>
+    <para>Make a note of the <option>access_key</option> and <option>secret_key</option>
+    as the secondary zones require them to authenticate with the master zone.</para>
+    <para>Add the system user to the master zone:</para>
+<screen>&prompt.cephuser;radosgw-admin zone modify --rgw-zone=us-east --access-key=<replaceable>ACCESS-KEY</replaceable> --secret=<replaceable>SECRET</replaceable>
+&prompt.cephuser;radosgw-admin period update --commit</screen>
+</sect3>
+
    <sect3 xml:id="ceph-rgw-fed-masterzone-updateperiod">
     <title>Update the Period</title>
     <para>
-     Because you changed the master zone configuration, you need to commit the
-     changes for them to take effect in the realm configuration structure.
-     Initially, the period looks like this:
+     After updating the master zone configuration, update the period:
     </para>
-<screen>&prompt.cephuser;radosgw-admin period get
-{
+<screen>&prompt.cephuser;radosgw-admin period update --commit</screen>
```

Comment:
We've just done a period update at the end of the last section. Maybe we leave the period update off there, and then mention now it's time to do a period update, and move to this section?